docs/template_plugin/src/template_plugin.cpp

```diff
@@ -186,13 +186,6 @@ InferenceEngine::QueryNetworkResult Plugin::QueryNetwork(const InferenceEngine::
 }
 // ! [plugin:query_network]
 
-// ! [plugin:add_extension]
-void Plugin::AddExtension(const InferenceEngine::IExtensionPtr& /*extension*/) {
-    // TODO: add extensions if plugin supports extensions
-    IE_THROW(NotImplemented);
-}
-// ! [plugin:add_extension]
```

Comment:
We need to update https://github.com/openvinotoolkit/openvino/tree/master/docs/IE_PLUGIN_DG where this snippet was used